### PR TITLE
wip

### PR DESCRIPTION
**Pull request by NullVoxPopuli**

### CODE REVIEW

File: packages/store/package.json [hunk 1]
```diff
@@ -31,6 +31,11 @@
       "default": "./addon/*.js"
     }
   },
+  "typesVersions": {
+    "*": {
+      "*": ["./unstable-preview-types/index.d.ts"]
```
Comment (NullVoxPopuli) [inline packages/store/package.json:36]: I noticed that these weren't built, so this won't work

File: packages/ember/src/-private/request.gts [hunk 2]
```diff
@@ -55,13 +57,13 @@ export class Request extends Component {
 
   <template>
     {{#if this.reqState.isLoading}}
```
Comment (NullVoxPopuli) [inline packages/ember/src/-private/request.gts:59]: named args must always come after positional args

File: packages/ember/src/-private/request.gts [hunk 3]
```diff
@@ -23,6 +23,8 @@ export class Throw extends Component {
   <template></template>
 }
 
+const and = (x: unknown, y: unknown) => x && y;
```
Comment (NullVoxPopuli) [inline packages/ember/src/-private/request.gts:26]: and wasn't defined or imported

See also: https://github.com/emberjs/rfcs/issues/816

**Comment by runspired** [comment]
thanks for poking around! got the branch working and applied the suggestions on gts usage there

**Pull request closed without merging**